src/main/src/jetty/database/ncdfgenerator/changelog.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet author="talend" id="add_sample_timeseries_data">
+        <!-- From anmn_ts_timeseries harvester -->
+        <sql>
+            CREATE TABLE ts_timeseries
```

Comment:
Yes its worth simplifying it .  I was just trying to get up and running to make the changes I needed so didn't do this.  I had already spent a lot of time doing this and it wasn't really a task we had identified.